### PR TITLE
Feature/assign consultation workflow

### PR DESCRIPTION
**Pull request by OwenGalvia**
Includes changes to consultation model

### CODE REVIEW

**Comment by taylorn01** [comment]
Needs version 0.4.2

**Comment by OwenGalvia** [comment]
> Needs version 0.4.2

done

**Review by taylorn01 — APPROVED** [review]
Looks good just want to double check that they don't already have important data created using the consultation model because it could make it difficult to update

**Comment by OwenGalvia** [comment]
@taylorn01 
0.6.4 has a clone of the model and uses that for this workflow. It works apart from the documentation step which frustratingly posts the tile correctly but gives a 500 claiming there is a key error 'nodegroup_id'. The documentation is added to the resource instance as expected, seems to be an issue with the actual workflow step.
I'm taking a break from this and moving to something else. I'll come back to it early next week unless you happen to know what the issue is.

File: coral/coral/pkg/graphs/resource_models/Consultation.json [hunk 1]
```diff
@@ -21,13 +21,13 @@
                     "graph_id": "8d41e49e-a250-11e9-9eab-00224800b26d",
                     "helpenabled": false,
```
Comment (taylorn01) [inline coral/coral/pkg/graphs/resource_models/Consultation.json:22]: Take a copy of this so we can use it in the future to replace the current consultation model on develop and then reset this back to how it is on develop

File: coral/coral/settings.py [hunk 2]
```diff
@@ -316,7 +316,7 @@ class GeoJSON:
 
 OAUTH_CLIENT_ID = ''  #'9JCibwrWQ4hwuGn5fu2u1oRZSs9V6gK8Vu8hpRC4'
 
-APP_VERSION = '0.3.2'
+APP_VERSION = '0.6.4'
```
Comment (taylorn01) [inline coral/coral/settings.py:319]: Make this match the version on develop and the auto updating will take effect

**Review by taylorn01 — CHANGES_REQUESTED** [review]

**Pull request merged**